translations/de/info/task_description.html

```diff
@@ -59,69 +57,69 @@
-        the world am I? Ah, THAT'S the great puzzle!' And she began
-        thinking over all the children she knew that were of the same age
-        as herself, to see if she could have been changed for any of
-        them.
+        Alice nahm den Fächer und die Handschuhe auf, und da der Gang sehr heiß war, fächelte
+        sie sich, während sie so zu sich selbst sprach: »Wunderbar! – wie seltsam heute Alles ist!
+        Und gestern war es ganz wie gewöhnlich. Ob ich wohl in der Nacht umgewechselt worden bin?
+        Laß mal sehen: war ich dieselbe, als ich heute früh aufstand? Es kommt mir fast vor,
+        als hätte ich wie eine Veränderung in mir gefühlt. Aber wenn ich nicht dieselbe bin,
+        dann ist die Frage: wer in aller Welt bin ich? Ja, das ist das Räthsel!« So ging sie
+        in Gedanken alle Kinder ihres Alters durch, die sie kannte, um zu sehen, ob sie in
+        eins davon verwandelt wäre.
     </p>
 
     <p>
-        "I'm sure I'm not Ada," she said, "for her hair goes in such
-        long ringlets, and mine doesn't go in ringlets at all; and I'm
-        sure I can't be Mabel, for I know all sorts of things, and she,
-        oh! she knows such a very little! Besides, SHE'S she, and I'm I,
-        and--oh dear, how puzzling it all is! I'll try if I know all the
-        things I used to know. Let me see: four times five is twelve,
-        and four times six is thirteen, and four times seven is--oh dear!
-        I shall never get to twenty at that rate! However, the
-        Multiplication Table doesn't signify:...
+        »Ich bin sicherlich nicht Ida,« sagte sie, »denn die trägt lange Locken, und mein
+        Haar ist gar nicht lockig; und bestimmt kann ich nicht Clara sein, denn ich weiß
+        eine ganze Menge, und sie, oh! sie weiß so sehr wenig! Außerdem, sie ist sie selbst,
+        und ich bin ich, und, o wie confus es Alles ist! Ich will versuchen, ob ich noch Alles
+        weiß, was ich sonst wußte. Laß sehen: vier mal fünf ist zwölf, und vier mal sechs
+        ist dreizehn, und vier mal sieben ist – o weh! auf die Art komme ich nie bis zwanzig!
+        Aber, das Einmaleins hat nicht so viel zu sagen; ...
     </p>
 
     <p style="float: right">
-        "Alice's Adventures in Wonderland." Lewis Carroll
+        "Alice's Abenteuer im Wunderland" von Lewis Carroll<br>
+        Aus dem Englischen von Antonie Zimmermann, 1869
     </p>
 
     <p style="clear: both;"></p>
 
 </div>
 
 <p>
-    After reading "Alice's Adventures in Wonderland,"
-    our robots decided to create their own "Multiplication table."
-    Stephan would lead this mission (yeah, that probably was a bad idea).
-    He forgot how to do multiplication and tried to invent a new method.
-    It’s a rather strange method if we may be so blunt.
+    Nach der Lektüre von "Alice im Wunderland ", haben sich unsere Roboter entschieden,
+    ihr eigenes Einmaleins zu erstellen. Stephan wollte diese Mission leiten
```

Comment:
_would lead this mission_ and _was a bad idea_
**was** ist Vergangenheit, man kann also annehmen das Stephan die Leitung bereits übernommen hat. Schliesslich besteht die Aufgabe ja darin Stephans Variante umzusetzten,
was wohl nicht nötig wäre, wenn Stephan nicht zuständig ist.

Der Kontext ist leider nicht ganz klar, was vermutlich daran liegt, dass Englisch nicht die Muttersprache von Bryukh ist. Solche Unstimmigkeiten sind mir bei seinen Aufgaben schon öfter aufgefallen. Hier ist es also an uns beiden eine brauchbare Lösung zu finden.

Wir können, denke ich, zwei Wege gehen:  
- Stephan _hat_ die Leitung übernommen **und** es _war_ ein schlechte Idee
**oder**
- Stephan _möchte_ die Leitung übernommen **und** es _ist_ eine schlechte Idee

Also ICH tendiere ja zu ersterem... :stuck_out_tongue_winking_eye: